include/range/v3/utility/static_const.hpp

```diff
@@ -14,14 +14,18 @@
 #define RANGES_V3_UTILITY_STATIC_CONST_HPP
 
 #include <range/v3/range_fwd.hpp>
+#include <range/v3/utility/inline_variable.hpp>
 
 namespace ranges
 {
     inline namespace v3
     {
         /// \ingroup group-utility
+        
         template<typename T>
-        struct static_const
+        struct
+        RANGES_DEPRECATED("Use RANGES_CXX_INLINE_VARIABLE instead which supports C++17 inline variables")
+        static_const
```

Comment:
Sorry about that, I didn't knew if `static_const` should be deprecated or not.  Thought it might be good to alert users that they could be using the macro instead, but `static_const` works just fine.